hack/make/release-deb

```diff
@@ -70,16 +92,14 @@ if [ ! -f "$APTDIR/conf/docker-engine-release.conf" ]; then
 	EOF
 fi
 
-# set the component for the version being released
-component="main"
-
-if [[ "$VERSION" == *-rc* ]]; then
-	component="testing"
-fi
-
-if [ $DOCKER_EXPERIMENTAL ] || [[ "$VERSION" == *-dev ]] || [ -n "$(git status --porcelain)" ]; then
-	component="experimental"
-fi
+for dir in contrib/builder/deb/*/; do
+	version="$(basename "$dir")"
+	codename="${version//debootstrap-}"
+	mkdir -p "$APTDIR/pool/$component/d/docker-engine/"
+	for arch in "${arches[@]}"; do
+		mkdir -p "$APTDIR/dists/$codename/$component/binary-$arch"
+	done
+done
```

Comment:
TBH I was having all kinds of trouble going from an empty directory to a populated repo and this was just one of the things I did to try to fix it. I'm not 100% sure it's necessary but I didn't feel it was a terrible change so I left it in. If it's genuinely unwanted I can work on getting rid of this chunk (and related) from the diff.